docs/source/considerations.rst

```diff
@@ -26,6 +26,32 @@ For more information simply refer to the
 `ØMQ guide <http://zguide.zeromq.org/page:all>`_.
 
 
+Closing connections
+===================
+
+For closing a specific connection from an agent we need to call the
+``close_socket()`` method, which takes the alias of the socket from the
```

Comment:
Maybe rename `close_socket` to `close`? (bind, connect, close; all are referring to sockets)